### ISSUE TITLE
Plánování - troja, volejbal, něco?

### ISSUE THREAD

**Issue by SmallhillCZ** [SmallhillCZ/bosan-cz#2]

**Comment by elenfant** [comment]
Bylo by super mít možnost pravidelných menších akcí, jako jsou schůzky/ježdění v Troji/volejbal, které by šly pro daný týden zrušit (ideálně rozsah platnosti od-do s případnými mezerami v termínech prázdnin apod.) a na které by se šlo přihlašovat tím simple klikáním...

**Comment by SmallhillCZ** [comment]
jo, já o tom dost uvažuju jak tohle řešit, ale zatím mě nenapadlo nic elegantního :(
